src/components/browse/sidebar/sidebar.tsx

```diff
@@ -5,43 +5,56 @@ import { Row, Col, Button } from "react-bootstrap";
 import { facetModel, facetFilterModel } from "../../../models/facets";
 import { searchResponseModel } from "../../../models/dataset";
 import { getDatasetsSearchResp } from "../../../api/browse";
+import PerfectScrollbar from 'react-perfect-scrollbar'
+import 'react-perfect-scrollbar/dist/css/styles.css';
```

Comment:
Lets download this css into the assets and use it from there. It freezes the version and also in future if this is not available from this url, we won't miss this css

Reply:
But it's not a CDN. When we install the libraries they are local in the container. We just use the files we downloaded from there. This is done before any calls to the back-end and not done with fron-facing users

Reply:
Ok, lets go with this solution.